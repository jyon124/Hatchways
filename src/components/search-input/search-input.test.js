describe("Search Input Component Tests", () => {
    it.todo("Renders a Search Input component");
    it.todo("Test if inputFunction invokes when input has been entered");
});
