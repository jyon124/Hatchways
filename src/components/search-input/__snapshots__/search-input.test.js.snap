// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search Input Component Tests Renders a Search Input component 1`] = `"<input class=\\"input\\" placeholder=\\"Search by name\\">"`;
